Makefile

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 The Kubernetes Authors.
```

Comment:
The previous PR had a [`deps` target](https://github.com/kubernetes/cloud-provider-gcp/pull/445/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R30-R33). Could that be included here?